ui/src/pages/GlobalHome.js

```diff
@@ -0,0 +1,293 @@
+        + 'use the product to help with a disaster response.'),
+  },
+  developersHowToHelp: {
+    id: 'GlobalHome.developersHowToHelp',
+    defaultMessage: ('You can help continue to improve Google Person Finder:'
+        + '<ul><li>Learn about the PFIF data model</li><li>Customize or '
+        + 'improve Person Finder</li></ul>'),
+    description: ('A summary of how software developers can help improve the '
+        + 'product.'),
+  },
+  getStarted: {
+    id: 'GlobalHome.getStarted',
+    defaultMessage: 'Get started',
+    description: ('Label on a button that takes users to a page explaining how '
+        + 'third-party developers can help.'),
+  },
+  howDoesItWork: {
+    id: 'GlobalHome.howDoesItWork',
+    defaultMessage: 'How does it work?',
+    description: ('Label on a button that takes users to a page explaining how '
+        + 'the product works.'),
+  },
+  learnHow: {
+    id: 'GlobalHome.learnHow',
+    defaultMessage: 'Learn how',
+    description: ('Label on a button that takes users to a page explaining how '
+        + 'third-party responders can help.'),
+  },
+  productNameWithEmphasis: {
+    id: 'GlobalHome.productNameWithEmphasis',
+    defaultMessage: '<b>Google</b> Person Finder',
+    description: ('Name of the product. Person Finder is a tool that helps '
+        + 'people reconnect with friends/family after a disaster.'),
+  },
+  responders: {
+    id: 'GlobalHome.responders',
+    defaultMessage: 'Responders',
+    description: ('A header for a section about how disaster responders, such '
+        + 'local officials, nonprofit organizations, etc., can use the product '
+        + 'to help.'),
+  },
+  respondersHowToHelp: {
+    id: 'GlobalHome.respondersHowToHelp',
+    defaultMessage: ('You can help people find each other in the aftermath of '
+        + 'a disaster:<ul><li>Embed Google Person Finder in your site</li>'
+        + '<li>Download data from Google Person Finder</li><li>Upload data '
+        + 'into Google Person Finder</li></ul>'),
+    description: 'A summary of how response organizations can use the product.',
+  },
+  tagline: {
+    id: 'GlobalHome.tagline',
+    defaultMessage: ('Reconnect with friends and loved ones in the aftermath '
+        + 'of natural and humanitarian disasters.'),
+    description: 'A one-line summary of what the product is for.',
+  },
+  youCanHelp: {
+    id: 'GlobalHomePage.youCanHelp',
+    defaultMessage: 'You can help',
+    description: ('Header on a section describing how people can help.'),
+  }
+});
+
+class GlobalHome extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoaded: false,
+      error: null,
+      repos: []
+    };
+  }
+
+  componentDidMount() {
+    fetch("/global/d/repoinfo")
+      .then(res => res.json())
+      .then(
+        (repos) => {
+          for (var i = 0; i < repos.length; i++) {
+            repos[i]['displayCat'] = i % 5;
```

Comment:
Optional: May be better to spell out "Cat" (category?)

You can also alternatively write this as:
```js
repos[i].displayCat = i % 5;
```